service/controller/v15/cluster_resource_set.go

```diff
@@ -40,10 +44,6 @@ type ClusterResourceSetConfig struct {
 func NewClusterResourceSet(config ClusterResourceSetConfig) (*controller.ResourceSet, error) {
 	var err error
 
-	if config.SSOPublicKey == "" {
-		return nil, microerror.Maskf(invalidConfigError, "%T.SSOPublicKey must not be empty", config)
-	}
```

Comment:
Boyscouting. The public key is not used on that level. The services down the road have to validate on their own in case they actually make use of it.